Yosemite/YosemiteTests/Stores/CustomerStoreTests.swift

```diff
@@ -0,0 +1,106 @@
+import XCTest
+@testable import Networking
+@testable import Yosemite
+
+class CustomerStoreTests: XCTestCase {
```

Comment:
I'm not very convinced on what should I test here at this point, or even if these tests are solid enough. For example, one of the problems I found is that if I attempt to dispatch a `CustomerAction` via the `CustomerStore` directly, or the app crashes, so either I've made this incorrectly or I'm testing the wrong thing 🤔 . Any feedback is appreciated, and happy to take a look during our pairing.